src/main/java/com/microsoft/azure/relay/ClientWebSocket.java

```diff
@@ -344,6 +341,15 @@ public void onError(Session session, Throwable cause) {
 		RelayLogger.throwingException(cause, this);
 	}
 	
+	private static ByteBuffer deepCopyByteBuffer(ByteBuffer original) {
```

Comment:
>deepCopyByteBuffer [](start = 27, length = 18)

No action needed for now, but if you ever find the need to do this same thing from another class let's put it into a helper class at that time.

ByteBufferUtil.deepCopy or similar (Util, Utility, Helper, *Ex)